python/instrumentation/openinference-instrumentation-bedrock/README.md

```diff
@@ -14,6 +14,9 @@ pip install openinference-instrumentation-bedrock
 
 ## Quickstart
 
+> **NOTE**:
+> OpenInference for AWS Bedrock supports both [`invoke_model`](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/services/bedrock-runtime/client/invoke_model.html#BedrockRuntime.Client.invoke_model) and [`converse`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/bedrock-runtime/client/converse.html#). For models that use the Messages API, such as Anthropic Claude 3 and Anthropic Claude 3.5, you can use the [Converse API](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_Converse.html) instead.
```

Comment:
this says "you can" is it the case that they must use the converse api for it to work properly? If so would adjust wording